packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/confirm/hosting-shared-confirm-settings.component.js

```diff
@@ -5,11 +5,13 @@ export default {
   controller,
   template,
   bindings: {
+    applyChanges: '<',
     goBack: '<',
-    rules: '<',
+    goToHosting: '<',
     model: '<',
-    success: '<',
-    cancel: '<',
+    oldModel: '<',
+    refresh: '<',
```

Comment:
prefer onRefresh and use & binding?

Reply:
hmm I've tried with `&` binding, but as it is passed through the routing resolve, it requires `<` type